File: lib/antlr-2.7.5/examples/csharp/heteroAST/CalcAST.cs
using System;
using antlr;
using antlr.collections;

public abstract class CalcAST : antlr.BaseAST {
	public abstract int Value();

	// satisfy abstract methods from BaseAST
	public override void initialize(int t, String txt) {
	}
	public override void initialize(AST t) {
	}
	public override void initialize(IToken tok) {
	}
}

File: lib/antlr-2.7.5/examples/java/heteroAST/CalcAST.java
import antlr.Token;
import antlr.collections.AST;

public abstract class CalcAST extends antlr.BaseAST {
	public abstract int value();

	// satisfy abstract methods from BaseAST
	public void initialize(int t, String txt) {
	}
	public void initialize(AST t) {
	}
	public void initialize(Token tok) {
	}
}
